guides/source/5_0_release_notes.md

```diff
@@ -355,9 +351,6 @@ Please refer to the [Changelog][action-mailer] for detailed changes.
 *   Template lookup now respects default locale and I18n fallbacks.
     ([commit](https://github.com/rails/rails/commit/ecb1981b))
 
-*   Template can use fragment cache like Action View template.
-    ([Pull Request](https://github.com/rails/rails/pull/22825))
-
```

Comment:
Why should this be removed? I thought it was added in https://github.com/rails/rails/commit/e40518f5d44303ed91641b342a53bdfb32753de3?